gcs/README.md

```diff
@@ -0,0 +1,246 @@
+# Google Cloud Storage Tasks
+
+These `Tasks` are for copying to and from GCS buckets from Pipelines.
+
+These `Tasks` do a similar job to the `GCS` `PipelineResource` and
+are intended as its replacement. This is part of our plan to [offer replacement
+`Tasks` for Pipeline Resources](https://github.com/tektoncd/catalog/issues/95)
+as well as
+[document those replacements](https://github.com/tektoncd/pipeline/issues/1369).
+
+## `gcs-download`
+
+A `Task` that fetches files or directories from a GCS bucket and puts them
+on a Workspace.
+
+### Workspaces
+
+* **credentials**: A workspace that contains a service account key as a JSON file.
+    This workspace should be populated from a Secret in your TaskRuns and PipelineRuns.
```

Comment:
might be useful to later include links to some docs about how to create these credentials